packages/devtools_app/lib/src/screens/inspector/diagnostics.dart

```diff
@@ -19,11 +19,11 @@ import 'inspector_controller.dart';
 import 'inspector_text_styles.dart' as inspector_text_styles;
 import 'inspector_tree.dart';
 
-final ColorIconMaker _colorIconMaker = ColorIconMaker();
-final CustomIconMaker _customIconMaker = CustomIconMaker();
-final CustomIcon defaultIcon = _customIconMaker.fromInfo('Default');
+final _colorIconMaker = ColorIconMaker();
+final _customIconMaker = CustomIconMaker();
+final defaultIcon = _customIconMaker.fromInfo('Default') as CustomIcon;
```

Comment:
why is this as check required?

Reply:
This is the one usage of this method. Make it non-nullable as the null path doesn't make sense anymore

Reply:
done